src/pudl/transform/ferc1.py

```diff
@@ -3546,6 +3644,64 @@ def process_xbrl_metadata(self, xbrl_metadata_json) -> pd.DataFrame:
         return pd.concat([meta, ending_balance])
 
 
+class ElectricitySalesByRateScheduleFerc1TableTransformer(
+    Ferc1AbstractTableTransformer
+):
+    """Transform class for :ref:`electricity_sales_by_rate_schedule_ferc1` table."""
+
+    table_id: TableIdFerc1 = TableIdFerc1.ELECTRICITY_SALES_BY_RATE_SCHEDULE_FERC1
+    has_unique_record_ids: bool = False
+
+    def add_axis_to_total_table_rows(self, df: pd.DataFrame):
+        """Add total to the axis column for rows from the total table.
+
+        Because we're adding the
+        sales_of_electricity_by_rate_schedules_account_totals_304 table into the mix,
+        we have a bunch of total values that get mixed in with all the _billed columns
+        from the individual tables. If left alone, these totals aren't labeled in any
+        way becuse they don't have the same _axis columns explaining what each of the
+        values are. In order to distinguish them from the rest of the sub-total data we
+        use this function to create an _axis value for them noting that they are totals.
+
+        It's worth noting that there are also some total values in there already.
+        Those would be hard to clean. The idea is that if you want the actual totals,
+        don't try and sum the sub-components, look at the actual labeled total rows.
+
+        This function relies on the ``sched_table_name`` column, so it must be called
+        before that gets dropped.
```

Comment:
This is mostly a clarifying question.. Does this mean that this whole table contains totals? it looks like you are overriding the full axis column for this table as well as the description. why both? is there information in either of those columns that your wiping out with this override?

Reply:
the description and axis columns are the same. The total table doesn't have either field, so we need to fill it in to be able to distinguish it from the non total values.